deploy/kubelet-config/03-kubelet-config.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   machineConfigPoolSelector:
     matchLabels:
-      managed.openshift.io/custom-kubelet: true
```

Comment:
Is this the real meat of the change, with other parts being auto-generated?

Reply:
Yes, it's the meat of the change. A problem is json representation of boolean causes some problems. The fix is to move away from boolean values, hence update to empty string.